ansible/roles/ocp4-workload-build-your-own-operator/tasks/remove_workload.yml

```diff
@@ -0,0 +1,35 @@
+---
+- environment:
+    KUBECONFIG: "{{ tmp_kubeconfig }}"
+  block:
+    - name: post_workload Tasks Complete
+      debug:
+       msg: "Pre-Software checks completed successfully - Removed"
+  
+    - name: Git clone the repo if it doesn't exist
+      command: git clone --branch v1.1 https://github.com/openshift-labs/lab-build-your-own-operator {{ tmp_git_location }}
```

Comment:
@jankleinert you changed the name of the branch from `v1.0` to `v1.1`.
I'll re-post my previous comment as it's still valid in this case:

> That's out of our process. We are using tags in agnosticd to pin a working state.
If you're using a branch you're bypassing this. If you push changes to this v0.1 branch, then deployments may fail.
I'm not against using files from an external repo, but can we use a commit ID instead of a branch ? That way the version would be pinned ?
(git clone + git checkout COMMITID)

Reply:
@fridim commented in chat as well, but to have the note here too, v1.1 is a tag. So this git clone --branch v1.1 <repo url> command is cloning that release now